ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -164,6 +164,21 @@ interface {{ vlan_interface.name }}
 {%     if vlan_interface.ipv6_access_group_out is arista.avd.defined %}
    ipv6 access-group {{ vlan_interface.ipv6_access_group_out }} out
 {%     endif %}
+{%     for ipv6_helper in vlan_interface.ipv6_helpers | arista.avd.natural_sort('ipv6_helper') %}
```

Comment:
I have not checked, but please verify that the order of the ipv6 helpers match the config ordering on EOS. (Both general order of commands on the SVI, but also if the order of individual helpers are sorted similar to natural_sort)

Reply:
In eos ipv6 helper addresses with no vrf defined are listed before those with a vrf defined.  To account for this I created two for loops which go through the ipv6 helper addresses. The first one checking for those without a vrf and the second one checking for those with a vrf. I think this is the best way to do this when factoring in simplicity and cost. Let me know if you have any other suggestions.